views/endorsement-page-sidebar.js

```diff
@@ -71,15 +74,35 @@ module.exports = (state, dispatch) => {
   `
 }
 
-const votedDifferentlyMessage = (measure) => {
+const endorsedPositionNotVote = (measure) => {
+  let previousVote = 'voted yes'
+  if (measure.vote_position === 'nay') { previousVote = 'voted no' }
+  if (measure.vote_position === 'abstained') { previousVote = html`abstained <span class="has-text-weight-normal">on</span>` }
```

Comment:
Would read: "abstained on on". Duplicated on, because lines 84 and 87 already have "on".